parsl/executors/high_throughput/interchange.py

```diff
@@ -160,6 +161,9 @@ def __init__(self,
 
         self.heartbeat_threshold = heartbeat_threshold
 
+        assert manager_selector is not None, "Undefined manager_selector"
```

Comment:
this probably shouldn't be allowed to be None here -- maybe you were getting a mypy error?

Reply:
Cutting this out in conjunction with removing the default value from the interchange init.

Reply:
Yeah, so I am getting an mypy error without this assertion, how should I address this?

Reply:
this was fixed by removing optionality...